Scripts/gen_CosmicStage2.sh

```diff
@@ -31,6 +32,7 @@ usage() {
   [ --low Resample 'Low' S1 output ]
```

Comment:
It looks like 'low' is an option, with default All, please add (opt) to the comment

Reply:
I don't think All is the default.
--low is the required option.